setup.py

```diff
@@ -12,12 +12,11 @@
     description="Corona Hakab Model Simulator",
     package_dir={"": "src"},
     packages=["corona_hakab_model", "corona_hakab_model_data"],
-    install_requires=["numpy", "scipy", "scikit-learn", "matplotlib"],
+    install_requires=["numpy", "scipy", "scikit-learn","matplotlib", "pytest"],
     extras_require={
         "quality": ["pytest", "tox", "flake8", "isort", "black"],
         "profiling": ["yappi"],
-        "pretty graphs": ["pyside2"],
-        "building": ["swimport"],
+        "pretty graphs": ["pyside2", "matplotlib"],
```

Comment:
matplotlib is already required above, we don't need it as an extra too

Reply:
done